### ISSUE TITLE
Feature request: entityschemas and validation.

### ISSUE THREAD

**Issue by dpriskorn** [LeMyst/WikibaseIntegrator#470]
support for entityschemas and validation seems to some degree to be supported in WBI. 
A user story could be:
- as a user I want to get an item from a wikibase and validate it against an entityschema in the same wikibase

**Comment by LeMyst** [comment]
You want to have your Entity and automatically validate against an entityschema, something like this :

```python
if my_entity.validate_schema('E345'):
  print('entity valid!)
else:
  print('try again 😬')
```

?

**Comment by dpriskorn** [comment]
Yes, that was my idea. Maybe given the distributed nature of LOD I want to validate against a schema in another wikibase?

**Comment by dpriskorn** [comment]
Here is gpl3 python code validating against an entityschema 😀
https://github.com/Teester/entityshape

**Comment by dpriskorn** [comment]
I created a alpha library for the entityshape API https://github.com/dpriskorn/PyEntityshape
